src/main/java/seedu/address/logic/commands/RemindCommand.java

```diff
@@ -21,26 +21,35 @@
 public class RemindCommand extends Command {
 
     public static final String COMMAND_WORD = "remind";
-    public static final String MESSAGE_REMIND_PERSON_WARNING = "";
-    public static final String MESSAGE_UNREMIND_PERSON_WARNING = "If you're trying to remove a Reminder for a Person, "
-            + "type \"remind INDEX\"";
-    public static final String MESSAGE_REMIND_PERSON_SUCCESS = "Created Reminder for Person: %1$s";
-    public static final String MESSAGE_UNREMIND_PERSON_SUCCESS = "Removed Reminder for Person: %1$s";
-    public static final String MESSAGE_EDIT_REMIND_PERSON_SUCCESS = "Edited Reminder for Person: %1$s";
+    public static final String MESSAGE_REMIND_PERSON_SUCCESS =
+            "Created Reminder for Client %1$s. He/she is added to the Reminders window.\n"
+            + "If the Reminders window is already open, close it and re-open it \n"
+            + "(via 'rm' command, 'Reminders' button from the 'File' tab, or press 'F4' key) to refresh the data!";
+    public static final String MESSAGE_UNREMIND_PERSON_SUCCESS =
+            "Removed Reminder for Client %1$s. He/She is removed the Reminders window.\n"
```

Comment:
Nice, I will make these changes for my commands too.

Reply:
👍